src/Particular.Analyzers.Tests/Cancellation/CatchAllShouldOmitOperationCanceledAnalyzerTests.cs

```diff
@@ -156,26 +156,34 @@ public async Task Bar()
 
         [Theory]
         [MemberData(nameof(CatchBlocks))]
-        public Task NotPassingToken(string catchBlocks)
+        public Task PassesNoTokenOrEmptyToken(string catchBlocks)
         {
-            const string DoesNotPassTokenTemplate =
-@"public class Foo
+            const string PassesNoTokenOrEmptyTokenTemplate =
+@"class Foo
 {
-    public async Task Bar(CancellationToken context)
+    async Task Bar(CancellationToken context)
     {
         var exOther = new Exception();
 
         try
         {
             await Test(42, true, 3.1415926);
+            await Test(42, true, 3.1415926, default);
+            await Test(42, true, 3.1415926, default(CancellationToken));
+            await Test(42, true, 3.1415926, CancellationToken.None);
+
+            await Test();
+            await Test(default);
+            await Test(default(ICancellableContext));
         }
         ##CATCH_BLOCKS##
     }
-    public Task Test(int i, bool b, double d, CancellationToken cancellationToken = default) => Task.CompletedTask;
+    Task Test(int i, bool b, double d, CancellationToken cancellationToken = default) => Task.CompletedTask;
+    Task Test(ICancellableContext cancellableContext = default) => Task.CompletedTask;
```

Comment:
This method can't be public because the `ICancellableContext` that the test infra injects into the code isn't public. I removed the other `public` modifiers while I was here because they are redundant.